### ISSUE TITLE
Console warnings are hard to read

### ISSUE THREAD

**Issue by kkga** [tw-in-js/twind#152]
I'm using twind on a project with Svelte+Rollup.
The warnings twind outputs into the console are hard to parse visually. I'm wondering if this is a problem of my setup or if it's a known issue and there are plans to improve it?

![image](https://user-images.githubusercontent.com/1460122/111954595-66232080-8af9-11eb-860f-de8d8abfb096.png)

**Comment by sastan** [comment]
I noticed that too. In chrome (where I implemented that feature) it looks _ok_:

![Chrome Warnings](https://user-images.githubusercontent.com/514405/111955829-4fc19880-8aea-11eb-8f39-e9bfc6abad7d.png)

We wanted to include the stack trace from the last twind function (tw, apply, or theme) to provide a place to go for fixing the issue. 

There is already an idea to refactor those messages to be more helpful by providing a link to `twind.dev/errors/[id]?params`.

If you have an idea how to provide better formatting please let me know. Here are the relevant source code links

- Modes: https://github.com/tw-in-js/twind/blob/main/src/twind/modes.ts
- Types: https://github.com/tw-in-js/twind/blob/main/src/types/twind.ts#L126

**Comment by kkga** [comment]
Oh, it does look much more readable in Chrome :+1:
My previous screenshot is from Firefox.

**Comment by rschristian** [comment]
As far as I can tell, `.stack` is a non-standard feature and has no on-going work to make it so. It's not really a viable way to produce such messages. [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/stack)

I'm a bit biased as I'm a Firefox user, but I'd be happy to drop the stack trace (at least for now) to instead get a readable error message. The current setup pollutes the console (especially when using external classes) and rarely makes it any faster to track down the issue. Every code editor worth mentioning has some way to search for strings, and being able to search my project for `h-scree` is much faster than trying to read that stack trace, find the problem file, the line number, then where along that line the issue is.

Safari, from my quick test, looks to be just as bad as FF in this regard, so it's not just a Chrome vs FF thing. 

And as a side note, FF + WMR creates a stack trace that only points to the root of the component due to the HTM transform. So users don't get the name of the broken rule or a (correct) line number. This isn't an issue with Twind in any way, as it's entirely FF's stack trace generation, but the combination of FF + WMR isn't even trading readability for a usable stack trace. Users get neither. 

![temp](https://user-images.githubusercontent.com/33403762/116326640-8d20e200-a78a-11eb-93d3-3a66e4bc3059.png)
![temp2](https://user-images.githubusercontent.com/33403762/116326642-8db97880-a78a-11eb-96ce-be532ab81776.png)

Stack trace shows the issue is on line 4, position 15, but that's the root of the HTM template. Contrived example obviously, but with 100+ lines in a component, this can be difficult.

**Comment by sastan** [comment]
I see the problem. I'll try to find a solution. Maybe that includes removing the stacktrace from the message. I'll keep you all updated.

**Comment by rschristian** [comment]
Sorry to poke, but any solution found / investigated? Unfortunately it's at the point where I have to fallback to [`patch-package`](https://www.npmjs.com/package/patch-package) or using a fork as the browser console is completely unusable alongside Twind. Too many valid messages get buried in the spam.

**Comment by sastan** [comment]
Hey, sorry for the delay. 

Which solution would you implement using patch-package? Would removing the stacktrace work for you?

**Comment by rschristian** [comment]
No worries. But pretty much just getting the stack trace out of there, yeah. Doesn't result in anything usable on Firefox

**Comment by sastan** [comment]
Release in 0.16.15
